Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.13.1-bullseye-slim
+FROM node:17-bullseye-slim
```

Comment:
Our organisation policy with Node is to use the latest LTS version which is 16 currently.